cfgov/v1/__init__.py

```diff
@@ -56,6 +56,9 @@ def parse_links(soup):
     icon_pattern = re.compile(settings.EXTERNAL_ICON_PATTERN)
     a_class = os.environ.get('EXTERNAL_A_CSS', 'icon-link icon-link__external-link')
     span_class = os.environ.get('EXTERNAL_SPAN_CSS', 'icon-link_text')
+    for tag in soup:
+        if hasattr(tag, 'style'):
+            del tag['style']
```

Comment:
So the only place I can think that we do use inline styles is in the hero https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/_includes/molecules/hero.html#L53. Will hero markup ever be going through this?